test/mailers/previews/registrations_preview.rb

```diff
@@ -0,0 +1,8 @@
+# Preview all emails at http://localhost:3000/rails/mailers/registrations
+class RegistrationsPreview < ActionMailer::Preview
```

Comment:
I like it to see how emails will look.
faster than sending it.
On Jun 9, 2015 8:55 PM, "GpoNsU" <notifications@github.com> wrote:

In test/mailers/previews/registrations_preview.rb
<https://github.com/TheTreeHouse/turniquette-api/pull/45#discussion_r32049760>
:

> @@ -0,0 +1,8 @@
> +# Preview all emails at http://localhost:3000/rails/mailers/registrations
> +class RegistrationsPreview < ActionMailer::Preview

previews on a API?

—
Reply to this email directly or view it on GitHub
<https://github.com/TheTreeHouse/turniquette-api/pull/45/files#r32049760>.